heroku.yml

```diff
@@ -7,4 +7,7 @@ release:
     - rake db:migrate
 run:
   web: bundle exec puma -C config/puma.rb
-
+  worker:
```

Comment:
Is it worth adding a `Procfile` to spin both these up with Foreman in development?

Reply:
It might be time to do that. I think it should be the responsibility of a separate PR rather than here though, we may have other opinions around it that may slow this mostly unrelated work down.